platforms/chibios/drivers/serial_usart.c

```diff
@@ -9,6 +9,21 @@
 
 #if defined(SERIAL_USART_CONFIG)
 static QMKSerialConfig serial_config = SERIAL_USART_CONFIG;
+#elif defined(MCU_AT32) /* AT32 MCUs */
+static QMKSerialConfig serial_config = {
+#    if HAL_USE_SERIAL
+    .speed = (SERIAL_USART_SPEED),
+#    else
+    .baud  = (SERIAL_USART_SPEED),
```

Comment:
I cant see support for SIO for this chip, so this else doesnt seem correct?

Reply:
Yeah, probably I just forgot to check and slip out of it for other chip needed. Already push a new commit to resolved this. Thanks!